tartufo/cli.py

```diff
@@ -122,6 +114,13 @@ def get_command(self, ctx: click.Context, cmd_name: str) -> Optional[click.Comma
     are excluded unless effectively excluded via the --include-path-patterns
     option.""",
 )
+@click.option(
+    "-of",
+    "--output-format",
+    type=click.Choice(["json", "compact", "text"]),
+    default="text",
+    help="Output can be in json/compact/text format. By default the output will be in text format.",
```

Comment:
I really like the description you added to the `README.md` for this option -- can you put that in here?